GVFS/GVFS.Service/Handlers/NotificationHandler.Mac.cs

```diff
@@ -21,7 +21,8 @@ public static NotificationHandler Instance
 
         public void SendNotification(ITracer tracer, int sessionId, NamedPipeMessages.Notification.Request request)
         {
-            throw new NotImplementedException();
+            // Log the notification till platform specific notifications become available.
+            tracer.RelatedInfo($"{request.Title} - {request.Message}.");

```

Comment:
Could your message include some information to say this is a notification message?